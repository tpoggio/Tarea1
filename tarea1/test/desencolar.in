encolar 15 4.0
esVaciaCola
desencolar
esVaciaCola
encolar 15 4.0
encolar 95 24.0
frente
desencolar
frente
encolar 15 14.0
frente
encolar 85 2.0
encolar 65 3.0
encolar 25 67.0
encolar 35 -12.0
encolar 75 2.5
encolar 45 2.1
encolar 55 -24.0
encolar 85 32.0
encolar 10 10.0
frente
desencolar
frente
Fin