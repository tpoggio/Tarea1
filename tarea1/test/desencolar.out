1>Encolar. 
2>La cola no está vacia. 
3>Desencolar. 
4>La cola está vacia. 
5>Encolar. 
6>Encolar. 
7>(15,4.00)
8>Desencolar. 
9>(95,24.00)
10>Encolar. 
11>(95,24.00)
12>Encolar. 
13>Encolar. 
14>Encolar. 
15>Encolar. 
16>Encolar. 
17>Encolar. 
18>Encolar. 
19>Encolar. 
20>Encolar. 
21>(95,24.00)
22>Desencolar. 
23>(15,14.00)
24>Fin.
